src/components/ButtonToggle/ControlledButtonToggle.tsx

```diff
@@ -0,0 +1,73 @@
+import React, {MutableRefObject, useRef} from 'react';
+import clsx from 'clsx';
+import './ButtonToggle.scss';
+import '../Button/Button.scss';
+import {Typography} from '../Typography';
+import {Loader} from '~/components/Loader';
+import type {ControlledButtonToggleProps} from './ButtonToggle.types';
+
+export const ControlledButtonToggle: React.ForwardRefRenderFunction<HTMLDivElement, ControlledButtonToggleProps> = ({
```

Comment:
You need to use React.forwardRef. and pass the `ref` as param. You can take a look at the Collapsible component as an example.